django/conf/global_settings.py

```diff
@@ -43,6 +43,11 @@ def gettext_noop(s):
 # If you set this to True, Django will use timezone-aware datetimes.
 USE_TZ = False
 
+# RemovedInDjango50Warning
+# Set True to continue using pytz tzinfo objects during the Django 4.x release
+# cycle.
+USE_DEPRECATED_PYTZ = False
```

Comment:
We should add deprecation warnings to the `Settings.__init__()`:
```python
        if self.is_overridden('USE_DEPRECATED_PYTZ'):
            warnings.warn(USE_DEPRECATED_PYTZ_DEPRECATED_MSG, RemovedInDjango40Warning)
```
and `UserSettingsHolder.__setattr__()`:
```python
        if name == 'USE_DEPRECATED_PYTZ':
            warnings.warn(USE_DEPRECATED_PYTZ_DEPRECATED_MSG, RemovedInDjango40Warning)
```
users should be warned even if they don't call time zones hooks.

Reply:
Yes, good. This is much neater 👍